src/modules/pages/components/ExpenditurePage/ExpenditurePage.tsx

```diff
@@ -82,10 +103,43 @@ const ExpenditurePage = () => {
         <div className={styles.mainContainer}>
           <main className={styles.mainContent}>
             <TitleDescriptionSection isEditable />
+            {loading ? (
```

Comment:
These 2 sections are almost identical. It's not good to repeat them. You can extract the repeating code to a variable or to a function (probably the latter, as it seems `isEditable` value will be different) and reuse it without repeating the code.
Also you need to change the html structure a but so as not to have `CommentInput` as a descendant of the `Form` component`, then you won't have the warning that Will mentioned.

Reply:
Thanks for comments. 
I removed `isEditable` variable and repetitions from the return statement.
To be honest, I don't know how can I change html structure to not have `CommentInput` as a descendant of the `Form` component. I need access to the form in all sections. Instead, I copied the `CommentInput` component and created a new one without the `form` tag in order not to nest them.

Reply:
There is no need in `SubformCommentInput` component. We can use the `CommentInput`. What you need to do is move `CommentInput` from the `LogSection` file to the `ExpenditurePage` and move/organise HTML structure so that it's not a descendant of the form. The inputs of your Form can be separate from the `CommentInput` form, and we treat comments differently. So it should all work.
Please, let me know if you still have any questions - happy to clarify.